tests/test-cases/Group1-Docker-Commands/1-03-Docker-Images.robot

```diff
@@ -30,19 +30,19 @@ Simple images
     ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} pull alpine:3.1
     Should Be Equal As Integers  ${rc}  0
     Should Not Contain  ${output}  Error
-    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} images
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} images alpine
```

Comment:
This is really fundamentally changing the nature of this test

Reply:
No, it doesn't.  This test pulls down 3 versions of alpines and checks if docker images shows 3 alpine images.  That's what the above does.